src/chrome/content/rules/ncar.cc.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+	MCB:
+		bbs.ncar.cc
+-->
+
+<ruleset name="ncar.cc">
+
+	<target host="bbs.ncar.cc" />
+	<exclusion pattern="^http://bbs\.ncar\.cc/$" />
+	<exclusion pattern="^http://bbs\.ncar\.cc/.+\.(html|php)" />
```

Comment:
From the looks of it, the tests are returning a false positive here, as they should have caught the unescaped dot in the exclusion pattern, cf. #7320.